Resources/Locale/en-US/mood/mood.ftl

```diff
@@ -51,4 +53,4 @@ mood-effect-RevolutionFocused = VIVA LA REVOLUTION!!!
 
 mood-effect-CultFocused = Dark Gods, grant me strength!
 
-mood-effect-TraitSanguine = I have nothing to worry about. I'm sure everything will turn out well in the end!
\ No newline at end of file
+mood-effect-TraitSanguine = I have nothing to worry about. I'm sure everything will turn out well in the end!
```

Comment:
```suggestion
mood-effect-TraitSanguine = I have nothing to worry about, I'm sure everything will turn out well in the end.
```

Reply:
that is not within the scope of this PR.